static/js/stream_settings_ui.js

```diff
@@ -431,8 +433,24 @@ export function render_left_panel_superset() {
     ui.get_content_element($("#manage_streams_container .streams-list")).html(html);
 }
 
+export function render_on_empty_streams(is_empty) {
```

Comment:
I think we can have a better function name here. One that I can think of is `render_empty_left_panel_message`.